main_flow/main_flow.py

```diff
@@ -47,13 +48,17 @@ def loadCache():
     if not (Path(file_instru).is_dir() and Path(file_ingri).is_dir()):
         return None
 
-    main_flow_logger.info('loading cache')
+    logging.info('loading cache')
```

Comment:
You didn't used the logger instance.
(I am not a reviewer I was just interested)

Reply:
I'm reverting the pr, the logger instance is deleted and the logging library will be back until the problem is fixed